scripts/find-testimonials.py

```
@@ -0,0 +1,791 @@
+    
+    print(f"Found {len(prs)} merged PRs")
+    return prs
+
+def get_comments_from_pr(pr_number, pr_url, pr_title):
+    """Fetch comments from a specific PR, excluding comments from EXCLUDE_USER."""
+    # Fetch both review comments and regular PR comments
+    comments_url = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/issues/{pr_number}/comments"
+    review_comments_url = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}/comments"
+    all_comments = []
+
+    for url_type, url in [("issue_comments", comments_url), ("review_comments", review_comments_url)]:
```

Ruff (0.11.9)

Loop control variable `url_type` not used within loop body

Rename unused `url_type` to `_url_type`

(B007)